x-pack/plugins/lens/public/indexpattern_datasource/help_popover.tsx

```diff
@@ -65,7 +65,6 @@ export const HelpPopover = ({
       isOpen={isOpen}
       ownFocus
       panelClassName="lnsHelpPopover__panel"
-      panelPaddingSize="none"
```

Comment:
Rather than removing the `panelPaddingSize` prop here and relying on negative margining to get the popover contents flush with the popover edges, it might be cleaner to just leave this `panelPaddingSize="none"` in place and instead put a `paddingSize="m"` prop on the `EuiPopoverTitle` component below. Not super important, but just thought it might be cleaner overall.